docs/interactions/structured-messages/webview.md

```diff
@@ -0,0 +1,26 @@
+# Webview
+
+The webview is a convenient way of opening a custom made webpage. It can be chosen by setting the optional target field to `webview` when sending a Structured Message containing a link element, such as [Rich Link](../rich-link), [Carousel](../carousel), or [Template](../template).
+
+The webview is a small window that sits atop the chat.
+
+# Design
```

Comment:
This is EM specific, we don't want this here. I guess it comes from the file move.

Let's do it like JB did on docs/interactions/structured-messages/webview-sdk.md and rename to a Engage Messaging Web example section.

Reply:
It's what I did in 7433c10